securedrop/server.py

```diff
@@ -230,6 +250,29 @@ async def add_contact(self, addc, stream):
         msg = self.users.add_contact(self.sock_to_email[stream], addc.name, addc.email)
         await self.write_status(stream, msg)
 
+    async def list_contacts(self, stream):
+        # three verification steps
+        current_user_email = self.sock_to_email[stream]
+        # 1: contacts_dict contains the names and email adresses that a user has added
+        contacts_dict = self.users.list_contacts(current_user_email)
+        contacts_dict_list = list(contacts_dict.keys())
```

Comment:
This isn't a dict, so the name is kind of confusing

Reply:
changed